README.md

```diff
@@ -165,7 +165,7 @@ jwt.verify(token, 'shhhhh', function(err, decoded) {
 
 // invalid token - synchronous
 try {
-  var decoded = jwt.verify(token, 'wrong-secret');
```

Comment:
Line 166 says this is an example for invalid token. One of the ways to achieve that is a wrong secret.